src/apiCalls.js

```diff
@@ -0,0 +1,35 @@
+const getBooksTittle = (bookTitle) => {
```

Comment:
The 'title' in the function name is spelled incorrectly here

Reply:
Goo catch Phil. I've fixed it already. Thank you